### ISSUE TITLE
ApiHeader default value not working

### ISSUE THREAD

**Issue by webhacking** [nestjs/swagger#672]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
I want to set a default value to my header property(requestId) using `ApiHeader` decorator.
So, i've set a value. but not showing my swagger page.

## Minimal reproduction of the problem with instructions

```typescript
import { applyDecorators } from '@nestjs/common';
import { ApiHeader } from '@nestjs/swagger';
import { v4 as uuidv4 } from 'uuid';

export function CommonHeaderReceiptDecorator() {
  return applyDecorators(
    ApiHeader({
      name: 'requestId',
      description: 'Definition of the requestId header A unique request ID, represented by a UUID.',
      required: true,
      allowEmptyValue: false,
      schema: {
        default: uuidv4()
      }
    }),
  );
}
```

## Environment

<pre><code>
Nest version: 6.14.2 
For Tooling issues:
- Node version: v13.12.0
- Platform: MAC

</code></pre>

**Comment by webhacking** [comment]
@kamilmysliwiec Also `example` method same thing

**Comment by kamilmysliwiec** [comment]
Please provide a minimum reproduction repository.

**Comment by kamilmysliwiec** [comment]
Unless I've missed something, there seems to be no `default` property in the official specification https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#parameterObject either.

**Comment by webhacking** [comment]
@kamilmysliwiec Okay thanks.

**Comment by theres** [comment]
@kamilmysliwiec https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#schemaObject `default` property  is defined on schema object level. Please reopen.

**Comment by kamilmysliwiec** [comment]
Fixed in 4.5.11
